test/extended/ldap_groups.sh

```diff
@@ -229,5 +229,14 @@ for (( i=0; i<${#schema[@]}; i++ )); do
 	openshift ex prune-groups --sync-config=sync-config.yaml --confirm
 	compare_and_cleanup valid_all_ldap_sync_prune.txt
 
+	# special-case test for ad-extended
```

Comment:
Doing that requires a lot of set-up that is done already in the loop... why can't we special-case it in the loop?